packages/@internationalized/number/src/NumberParser.ts

```diff
@@ -180,34 +263,103 @@ class NumberParserImpl {
   }
 
   sanitize(value: string) {
+    let sanitizedValue = value.trim();
+
+    let numeralMatches = sanitizedValue.match(NUMERALS_REGEX);
+
+    if (numeralMatches) {
```

Comment:
It depends on the locale. `.925` becomes `0.925` in locales that use `.` for the decimal. However, in locales using `,` for groups and `.` for decimal, `,925` will be interpreted as `925`, because with 3 digits after the symbol, the ambiguous comma is interpreted as a grouping symbol. Where there is no ambiguity `,92` and `,9253` get interpreted as `0.92` and `0.9253`.

Reply:
I may have fixed this with: [74c7704](https://github.com/adobe/react-spectrum/pull/6520/commits/74c7704054b00905d71a9b3d87718334c64ff2e0)